include/gsl/span

```diff
@@ -355,6 +355,13 @@ namespace details
         constexpr extent_type(size_type size) { Expects(size == Ext); }
 
         constexpr size_type size() const noexcept { return Ext; }
+
+    private:
```

Comment:
Since this is only used in debug mode for the sake of the natvis, I suggest wrapping this section in a _DEBUG check.
```suggestion
    private:
#if defined(_DEBUG)
```